cmd/apps/istio_app.go

```diff
@@ -28,119 +30,136 @@ func MakeInstallIstio() *cobra.Command {
 		Example:      `  arkade install istio --loadbalancer`,
 		SilenceUsage: true,
 	}
-	istio.Flags().Bool("update-repo", true, "Update the helm repo")
-	istio.Flags().String("namespace", "istio-system", "Namespace for the app")
-	istio.Flags().Bool("init", true, "Run the Istio init to add CRDs etc")
+	istio.Flags().StringP("version", "v", istioVer, "Specify a version of Istio")
+	istio.Flags().StringP("manifests", "d", "https://github.com/istio/istio/releases/download/1.9.0/istio-1.9.0-linux-amd64.tar.gz", "Specify a path to a directory of charts and profiles")
```

Comment:
Would it make sense for you to template this with the version you set in `var istioVer = "1.9.1"`?

Reply:
We don't need to set that path as istioctl applies default URL as per the architecture. I am making changes to apply manifest only if user provides it.

Reply:
I feel like you should remove the default use-case in that case.